doc/rbd/rbd-mirroring.rst

```diff
@@ -111,13 +116,13 @@ Bootstrap Peers
 ---------------
 
 In order for the ``rbd-mirror`` daemon to discover its peer cluster, the peer
-needs to be registered to the pool and a user account needs to be created.
+needs to be registered and a user account needs to be created.
```

Comment:
What do you think about changing "needs to be" to "must"?

I know this is beyond the scope of this change, but I just want an opinion from another writer.

Reply:
You aren't wrong.  I made the same edit elsewhere too.